app/app/explore/pools/PoolsView.tsx

```diff
@@ -0,0 +1,19 @@
+"use client";
+
+import { PoolDetails, StatsData } from "@/midgard";
+import LiquidityPools from "@/app/explore/pools/LiquidityPools";
+import ExploreNav from "../components/ExploreNav";
+
+interface PoolsViewProps {
+  pools: PoolDetails;
+  stats: StatsData;
+}
+
+export default function PoolsView({ pools, stats }: PoolsViewProps) {
+  return (
+    <main className="mx-16">
+      <ExploreNav />
```

Comment:
im wondering if it's possible to move this into a layout.tsx file under explore directory. if possible, then you can get rid of this the pools and savers view components and simply render the LiquidityPools and Savers components within their respective server components. the only thing im not sure about is if the [id] pages will inherit the explore layout but it might be fine if we don't include a layout.tsx file there.